### PR TITLE
[SDK-3913] Move react beta quickstart to GA

### PR DESCRIPTION
**Pull request by ewanharris**
Remove the beta react QS and updates the GA content to be ready for 2.0.0 release

### CODE REVIEW

File: articles/quickstart/spa/react/01-login.md [hunk 1]
```diff
@@ -46,7 +48,7 @@ The `Auth0Provider` component takes the following props:
 
 <%= include('../_includes/_auth_note_custom_domains') %>
 
-- `redirectUri`: The URL to where you'd like to redirect your users after they authenticate with Auth0.
+- `authorizationParams.redirectUri`: The URL to where you'd like to redirect your users after they authenticate with Auth0.
```
Comment (frederikprijck) [inline articles/quickstart/spa/react/01-login.md]:
```suggestion
- `authorizationParams.redirect_uri`: The URL to where you'd like to redirect your users after they authenticate with Auth0.
```

File: articles/quickstart/spa/react/01-login.md [hunk 2]
```diff
@@ -76,7 +78,7 @@ export default LoginButton;
 :::panel Checkpoint
 Add the `LoginButton` component to your application. When you click it, verify that your React application redirects you to the [Auth0 Universal Login](https://auth0.com/universal-login) page and that you can now log in or sign up using a username and password or a social provider.
 
-Once that's complete, verify that Auth0 redirects you to your application using the value of the `redirectUri` that you used to configure the `Auth0Provider`.
+Once that's complete, verify that Auth0 redirects you to your application using the value of the `authorizationParams.redirectUri` that you used to configure the `Auth0Provider`.
```
Comment (frederikprijck) [inline articles/quickstart/spa/react/01-login.md]:
```suggestion
Once that's complete, verify that Auth0 redirects you to your application using the value of the `authorizationParams.redirect_uri` that you used to configure the `Auth0Provider`.
```

File: articles/quickstart/spa/react/02-calling-an-api.md [hunk 3]
```diff
@@ -39,9 +39,11 @@ ReactDOM.render(
   <Auth0Provider
     domain="${account.namespace}"
     clientId="${account.clientId}"
-    redirectUri={window.location.origin}
-    audience="https://${account.namespace}/api/v2/"
-    scope="read:current_user update:current_user_metadata"
+    authorizationParams={{
+      redirectUri: window.location.origin,
```
Comment (frederikprijck) [inline articles/quickstart/spa/react/02-calling-an-api.md]:
```suggestion
      redirect_uri: window.location.origin,
```

File: articles/quickstart/spa/react/interactive.md [hunk 4]
```diff
@@ -43,7 +43,7 @@ For the SDK to function properly, you must set the following properties in the A
...
 - `clientId`: The ID of the Auth0 Application you set up earlier in this quickstart. You can find this in the Auth0 Dashboard under your Application's Settings in the Client ID field.
-- `redirectUri`: The URL in your application that you would like Auth0 to redirect users to after they have authenticated. This corresponds to the callback URL you set up earlier in this quickstart. You can also find this value in the Auth0 Dashboard under your Application's Settings in the Callback URLs field. Make sure what you enter in your code matches what you set up earlier or your users will see an error.
+- `authorizationParams.redirectUri`: The URL in your application that you would like Auth0 to redirect users to after they have authenticated. This corresponds to the callback URL you set up earlier in this quickstart. You can also find this value in the Auth0 Dashboard under your Application's Settings in the Callback URLs field. Make sure what you enter in your code matches what you set up earlier or your users will see an error.
```
Comment (frederikprijck) [inline articles/quickstart/spa/react/interactive.md]:
```suggestion
- `authorizationParams.redirect_uri`: The URL in your application that you would like Auth0 to redirect users to after they have authenticated. This corresponds to the callback URL you set up earlier in this quickstart. You can also find this value in the Auth0 Dashboard under your Application's Settings in the Callback URLs field. Make sure what you enter in your code matches what you set up earlier or your users will see an error.
```

File: articles/quickstart/spa/react/interactive.md [hunk 5]
```diff
@@ -83,13 +83,13 @@ You should now be able to log in or sign up using a username and password.
 Click the login button and verify that:
 * your React Application redirects you to the Auth0 Universal Login page
 * you can log in or sign up
-* Auth0 redirects you to your application using the value of the `redirectUri` you used to configure the `Auth0Provider`
+* Auth0 redirects you to your application using the value of the `authorizationParams.redirectUri` you used to configure the `Auth0Provider`
```
Comment (frederikprijck) [inline articles/quickstart/spa/react/interactive.md]:
```suggestion
* Auth0 redirects you to your application using the value of the `authorizationParams.redirect_uri` you used to configure the `Auth0Provider`
```

File: articles/quickstart/spa/react/interactive.md [hunk 6]
```diff
@@ -83,13 +83,13 @@ You should now be able to log in or sign up using a username and password.
...
 
 :::
 
 :::checkpoint-failure
 Sorry about that. Here's a couple things to double check:
-* you configured the correct `redirectUri`
+* you configured the correct `authorizationParams.redirectUri`
```
Comment (frederikprijck) [inline articles/quickstart/spa/react/interactive.md]:
```suggestion
* you configured the correct `authorizationParams.redirect_uri`
```

**Review by frederikprijck — CHANGES_REQUESTED** [review]

File: articles/quickstart/spa/react/01-login.md (hunk 1 above)
Comment (ewanharris) [reply articles/quickstart/spa/react/01-login.md]: Doh, thanks for the catch!

**Review by frederikprijck — APPROVED** [review]

**Pull request merged**